examples/notebooks/responsibleaidashboard-diabetes-decision-making.ipynb

```diff
@@ -953,7 +902,7 @@
     "    type: command\n",
     "    component: azureml:rai_score_card:{version_string}\n",
     "    inputs:\n",
-    "      dashboard: ${{parent.jobs.gather_01.outputs.dashboard}}\n",
+    "      dashboard: ${{{{parent.jobs.gather_01.outputs.dashboard}}}}\n",
```

Comment:
@kicha0 Please note this if you add the scorecard to any other YAMLs. I'm not sure why the cell didn't fail the notebook, because it should have done.